packages/salesforcedx-vscode-apex-debugger/src/messages/i18n.ts

```diff
@@ -19,5 +19,9 @@ export const messages = {
   select_exception_text: 'Select an exception',
   select_break_option_text: 'Select break option',
   always_break_text: 'Always break',
-  never_break_text: 'Never break'
+  never_break_text: 'Never break',
+  are_you_still_debugging_text:
+    'We noticed that you have been idle for awhile. Are you still debugging?',
```

Comment:
Shown at the top of VS Code when the idle-session timer is triggered:

![idleprompt](https://user-images.githubusercontent.com/7286437/32973362-d75d3828-cbab-11e7-8fce-bce5ea93ad23.png)

Reply:
"awhile" should be "a while" (http://www.quickanddirtytips.com/education/grammar/while-versus-awhile)